styleguide/components/31-molecules/teaser/teaser.config.js

```diff
@@ -27,36 +27,34 @@ module.exports = {
       name: 'article',
       context: {
         preview: '@preview',
-        teaser_uid: 'uid',
-        teaser_type: 'teaser-article',
-        teaser_paragraph_text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam corporis cumque debitis eligendi harum iste laborum quibusdam ut, vitae voluptate?',
-        teaser_title: 'General article title text',
+        type: 'teaser-article',
+        paragraph_text: 'Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam corporis cumque debitis eligendi harum iste laborum quibusdam ut, vitae voluptate?',
+        title: 'General article title text',
         teaser_published_date: 'Gepost op Zaterdag 7 januari 2019',
-        teaser_link_text: 'read more',
-        teaser_link_link: '#',
-        teaser_image_src: 'https://via.placeholder.com/800x500&text=8:5+(800x500)',
-        teaser_image_alt_text: 'alt text',
+        link_text: 'read more',
+        link: '#',
+        image_src: 'https://via.placeholder.com/800x500&text=8:5+(800x500)',
+        image_alt_text: 'alt text',
         teaser_tag: 'Just some tag'
```

Comment:
There are still some variables in here that are prefixed with `teaser_`. 

(sorry for the nitpicking 🤐)